Commit for creating a PR with two introductory connector topics (#597)
mulesoft/docs-connectors: modules/ROOT/pages/introduction/intro-use-exchange.adoc

= Use Anypont Exchange to Discover Connectors, Templates, and Examples

> 

Reviewer: Metadata cannot be in a new line. Remove the extra line.
```suggestion
```
Also, don't specify `:imagesdir: ../../assets/images/`. You don't need it and you might be overwriting the actual imagesdir declaration from the `_attributes.adoc[]` file that you are including.

For more information about searching for assets in Exchange, see xref:exchange::to-find-info.adoc[Find Asset Information].

[[discover-add-connectors]]
== Discover and Add Connectors to Flow Designer and Studio Projects

Discover connectors either by starting the search in Exchange or by accessing Exchange from within an Anypoint Flow Designer or Anypoint Studio project:

* Start in Exchange to view a list of all or a subset of MuleSoft-provided connectors. From there, you can view a connector in Flow Designer or download it to Studio.
* Access Exchange from within a Flow Designer or Studio project to locate a template applicable to your use case.

=== Discover Connectors Provided by MuleSoft

To use Exchange as a starting point for discovering all or a subset of MuleSoft-provided connectors:

. https://www.mulesoft.com/exchange/[Access Exchange].
. Under *All assets* on the left side of the screen, click *Provided by MuleSoft*.
. Select *Connectors* in the drop-down list to the left of the search bar.
. To filter the list of connectors, add text to the search field.
+
For example, if you enter `Salesforce`, the search returns all Salesforce connectors.
+

> . Click on connector to view its details.

Reviewer 2: Click on **a** or click on **the** connector...

[[add-connector-to-flow-designer]]
=== Add a Connector to a Flow Designer App

> When you are working in a Flow Designer project, you can add a connector directly to the project. You can use Flow Designer only for supported connectors.

Reviewer 2: ```suggestion
When you are working in a Flow Designer project, you can add a connector directly to the project. You can use Flow Designer for only supported connectors.
```

> . Log into Anypoint Platform.

Reviewer 2: ```suggestion
. Log in to Anypoint Platform.
```

. Click the connector name in *Available modules*.
. Click *Add*.
. Click *Finish*.

=== Next Step

Follow the directions in the connector's User Guide for configuring the connector in Design Center (using Flow Designer) or Studio.

[[discover-add-templates]]
== Discover and Add Templates to Flow Designer or Studio

A template is a pre-built project that you can use as a starting point for applicable use cases. Many templates include one or more connector configurations.

Discover templates by starting in Exchange or by accessing Exchange from within Flow Designer or Studio:

* Start in Exchange to view a list of all or a subset of MuleSoft-provided templates. From there, you can view a template in Flow Designer or download it to Studio.
* Access Exchange from within Flow Designer or Studio to locate a template that fits your use case.

When you locate a template in Exchange, the landing page of the associated connector contains:

* An overview of what the template does
* Prerequisites for using the template
* Instructions for configuring the template

=== Discover MuleSoft Templates

To use Exchange as a starting point for discovering MuleSoft-provided templates:

> . https://www.mulesoft.com/exchange/[Log into Exchange].

Reviewer 2: ```suggestion
. https://www.mulesoft.com/exchange/[Log in to Exchange].
```

Discover examples by starting in Exchange or by accessing Exchange from within a Studio project:

* Start in Exchange to view a list of all or a subset of MuleSoft-provided examples. From there, you can view download an example to Studio.
* Access Exchange from Flow Designer or Studio to locate an example that fits your project's use case.

When you locate an example in Exchange, the landing page explains how to set up and run the example.

=== Discover MuleSoft Examples

To use Exchange as a starting point for discovering MuleSoft-provided examples:

. Log in to Exchange with your Anypoint username and password.
. Under *All assets* on the left side of the screen, click *Provided by MuleSoft*.
. Select *Examples* in the drop-down list to the left of the search bar.
. To narrow the list of examples, enter applicable text in the Search field.
+
For example, if you enter `Salesforce Connector`, the search returns all examples that use the Salesforce Connector.
+
. Click on an example to view its details.

> . To download the example for use in Studio, click *Download* on the upper right of the screen.

Reviewer 2: ```suggestion
. To download the example for use in Studio, click *Download* on the upper-right of the screen.
```

+
The system downloads the file as a `.jar` file.

=== Import a Downloaded Example to Studio

If you downloaded an example from Exchange, you can import the downloaded `.jar` file to Studio to create the new project:

. From Studio, select *File > Import*.
. In the Import wizard, expand the *Anypoint Studio* folder.
. Select *Packaged mule application (.jar)* and click *Next*.
. In the *File* field, optionally enter a name for the file.
. Click the ellipses (…​) button.
. Select the `.jar` file to import.
. Click *Open*.
+
Studio imports the example as a new Studio project.

=== Add an Example Directly to Studio

When you are working in Studio, you can import an example directly from Exchange:

. From Studio, click the Exchange icon *(X)* in the upper-left of the Studio taskbar.

> . Log into Exchange using your Anypoint Platform username and password.

Reviewer 2: ```suggestion
. Log in to Exchange using your Anypoint Platform username and password.
```
